recipes/conda-bash-completion/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "conda-bash-completion" %}
+{% set version = "1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: "{{ version }}"
+  url: https://github.com/tartansandal/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 82fa38dae6cec42095b5a1f82d5db41b35a7a2c35642a9b7ad548385ae601659
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - bash-completion
```

Comment:
The `build` section is for build tooling. I think you want `host` here.

Reply:
If you don't need anything from the bash-completion package when building, then this section should be removed all together.

Reply:
The `bash-completion` dependency was to get the `$PREFIX/share/bash-completion/completions/` directory. But making that independently is much simpler :smile:

Reply:
Fixed :-)